packages/examples/src/pages/PixelCanvas/components/Pixel/Pixel.css

```diff
@@ -0,0 +1,10 @@
+.pixel {
```

Comment:
This is global css, what is actually bad practice
rework to  css-modules + bem notation as in the comment  above

`.Pixel` and `.Pixel:hover`